jraft-core/src/main/java/com/alipay/sofa/jraft/Node.java

```diff
@@ -226,4 +227,32 @@
      * @throws LogIndexOutOfBoundsException  the special index is out of bounds.
      */
     UserLog readCommittedUserLog(final long index);
+
+    /**
+     * SOFAJRaft End User can implement the ReplicatorStateListener interface by themselives
+     * So users can do their own logical operator in this listener when replicator started ,stopped or has some errors
```

Comment:
started ,stopped -> started, stopped

Reply:
Okay,I will fix this codes comments and then push it again.Thanks!

Reply:
started -> is created, stopped-> is  destroyed.